From 13743762632cdd6b242edc08fa8db4250c829325 Mon Sep 17 00:00:00 2001
From: Jiyong Park <jiyong@google.com>
Date: Mon, 3 Jul 2017 21:38:32 +0900
Subject: [PATCH 3/3] Build libstagefrighthw with BOARD_VNDK_VERSION

Instead of relying on the global include paths, use the new headers lib
'media_plugin_headers'.

Bug: 37342627
Test: BOARD_VNDK_VERSION=current m -j libstagefrighthw
Change-Id: I09fc6c053a0b18c6a0c8f6d017bc001d50d11ca2
---
 libstagefrighthw/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libstagefrighthw/Android.mk b/libstagefrighthw/Android.mk
index a14c11bf2..a7f6e860f 100644
--- a/libstagefrighthw/Android.mk
+++ b/libstagefrighthw/Android.mk
@@ -41,6 +41,9 @@ LOCAL_SHARED_LIBRARIES :=       \
         libdl                   \
         libui                   \
 
+LOCAL_HEADER_LIBRARIES := \
+        media_plugin_headers
+
 LOCAL_MODULE := libstagefrighthw
 
 LOCAL_VENDOR_MODULE := true
-- 
2.17.1

